src/v3/src/components/AuthContainer/AuthContainer.tsx

```diff
@@ -22,6 +22,7 @@ const AuthContainer: FunctionComponent = ({ children }) => {
     // @ts-ignore OKTA-471233
     <Box
       as="main"
+      id="okta-sign-in"
```

Comment:
This was needed because when OK-PL was used as the language, there is some logic in main.ts (playground) that searches for the main component with this ID value. if it doesnt find it, it causes a js error that prevents you from progressing forward. 

That logic also verifies and asserts there are no unlocalized strings in the current view.